clap_derive/tests/global.rs

```diff
@@ -0,0 +1,64 @@
+use clap::Clap;
+
+#[test]
+fn propagate_global_arg_in_subcommand_to_subsubcommand_2053() {
```

Comment:
Let's move this to `tests/global_args.rs` since this issue is not derive specific.

Reply:
Done.